src/clojure_ttt/presenter.clj

```diff
@@ -0,0 +1,29 @@
+(ns clojure-ttt.presenter
+  (use [clojure-ttt.board]))
+
+(defn- clear-console []
+  (println "\033[H\033[2J"))
+
+(defn- welcome-user []
+  (println "\033[35mWelcome! Please select a gamemode.")
+  (println "------------------------------------------"))
+
+(defn show-gamemode-options []
+  (clear-console)
+  (welcome-user)
+  (println "\033[36mType 0 to play human vs human. \n1 to play vs a computer. \n2 to watch two computers play."))
```

Comment:
This was not clear when displayed in the console.  Probably because of the word 'type' being either a verb or a noun.. You could lose 'type' altogether and have:
0 - to play human vs human
1  - to play ....